app/src/test/java/org/mozilla/fenix/components/StoreProviderTest.kt

```diff
@@ -24,7 +24,7 @@ class StoreProviderTest {
 
     private class BasicState : State
 
-    private val basicStore = Store(BasicState()) { state, _: Action -> state }
+    private val basicStore = Store(BasicState(), { state, _: Action -> state })
```

Comment:
Oh, sorry, I added a parameter to the store constructor and didn't expect that to cause any issues. Didn't think of _trailing lambda_ usage.

Reply:
All good @pocmo, it's a weird edge case.